src/components/directorybrowser/directorybrowser.js

```diff
@@ -87,7 +87,7 @@ function getEditorHtml(options, systemInfo) {
     let html = '';
     html += '<div class="formDialogContent scrollY">';
     html += '<div class="dialogContentInner dialog-content-centered" style="padding-top:2em;">';
-    if (!options.pathReadOnly) {
+    if (!options.pathReadOnly && (options.instruction || systemInfo.OperatingSystem)) {
```

Comment:
The operating system will never be set as it was removed from the system info in 10.9:

https://github.com/jellyfin/jellyfin/pull/9175